src/libtriton/ast/ast.cpp

```diff
@@ -3122,435 +3122,387 @@ namespace triton {
 namespace triton {
   namespace ast {
 
+    void checkNode(AbstractNode* node){
```

Comment:
Sure, static should be in. But  may I don't use inline. It shall cause too many repeat code in binary.